pkg/api/vlabs/validate.go

```diff
@@ -16,13 +16,32 @@ import (
 var (
 	validate        *validator.Validate
 	keyvaultIDRegex *regexp.Regexp
+	// Any version has to be mirrored in https://acs-mirror.azureedge.net/github-coreos/etcd-v[Version]-linux-amd64.tar.gz
+	etcdValidVersions = [...]string{"2.5.2", "3.2.0"}
```

Comment:
Any reason why you didn't do 3.2.9?

Reply:
Just remaining on a older side of releases. The new versions bring new functions and bugs that takes time to iron out.

Reply:
patch releases are more stable. Minors may not be. If you're going for stability then go for the latest 3.1 patch release. You're on the edge with 3.2 anyways. I generally stay a minor behind unless there's something pressing.

Reply:
Alright will go for 3.1.